src/commands/quickLinksCommand.ts

```diff
@@ -76,6 +76,12 @@ export class QuickLinksCommand extends Command {
         "WHF is short for 'Well hello friends', the greeting used by Andreas in his coding videos",
       help: "Explains the meaning of 'whf'",
     },
+    {
+      name: "wpt",
+      response: `Charts comparing our score to other browsers, over time: <https://linegoup.lol>
+Our latest WPT results: <https://wpt.fyi/results/?product=ladybird>`,
```

Comment:
Honestly not sure if this will correctly produce a message with two lines.

Reply:
We'll see!